packages/cozy-sharing/src/components/ShareRecipientsInput.jsx

```diff
@@ -49,25 +47,19 @@ const ShareRecipientsInput = ({
   }
 
   return (
-    <div>
-      <label className={styles['coz-form-label']} htmlFor="email">
-        {label}
-      </label>
```

Comment:
Can you explain why you removed that one?

Reply:
UI changed and there is no label here anymore. Do I need to explain it in the commit message?